src/implementations/vanilla/components/banner/_banner.scss

```diff
@@ -184,18 +189,12 @@ $_font-weight: null !default;
 .ecl-banner--text-box {
   background: map.get($_background, 'text-box');
 
+  // manage ratio 4:1
   .ecl-banner__image {
+    aspect-ratio: 4/1;
```

Comment:
this aspect ratio overrides the one from the banner size, so changing size does not work anymore